core/contracts/oracle/implementation/test/GovernorTest.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.6.0;
+
+pragma experimental ABIEncoderV2;
+
+import "../Governor.sol";
+
+
+// GovernorTest exposes internal methods in the Governor for testing.
+contract GovernorTest is Governor {
```

Comment:
Pity that you have to make a whole extra contract to test internal logic like this.

Reply:
Yeah :/